src/app/containers/PageHandlers/withPageWrapper/index.test.jsx

```diff
@@ -13,6 +14,8 @@ describe('with pageWrapper', () => {
   const PageWrapperHOC = WithPageWrapper(PageWrapperContainer);
   shouldMatchSnapshot(
     `should render correctly`,
-    <PageWrapperHOC {...dataProps} />,
+    <ServiceContextProvider service="news">
```

Comment:
I don't think we need to be wrapping the PageWrapper with the service context. The PageWrapper receives the service name via a prop on the data object.

Whats the error that this was solving?

Reply:
i ran it locally, and it seems to fix these warnings :
![image](https://user-images.githubusercontent.com/30599794/63087302-badbde00-bf49-11e9-85be-f76081f52792.png)
![image](https://user-images.githubusercontent.com/30599794/63087313-c0392880-bf49-11e9-934d-ea1b44d8a2ed.png)

Reply:
Ok this is fine then. In the app the PageWrapper is a child of the withContexts provider so this makes sense.